docs/ConfigurationGuide.md

```diff
@@ -85,15 +88,57 @@ Several functions in ACAPy VC-AuthN can be tweaked by using the following enviro
 | LOG_TIMESTAMP_FORMAT      | string                                 | determines the timestamp formatting used in logs                                                                                                                                                                                                                                                                                                                                                                                                       | Default is "iso"                                                                                                                        |
 | LOG_LEVEL                 | "DEBUG", "INFO", "WARNING", or "ERROR" | sets the minimum log level that will be printed to standard out                                                                                                                                                                                                                                                                                                                                                                                        | Defaults to DEBUG                                                                                                                       |
 
-### Multi-Tenant Webhook Registration
+### Legacy Variable Support
```

Comment:
Understand keeping these around for backwards compatibility, but am understanding right that the Multitenancy didn't really work until @MonolithicMonk 's recent fixes? In that case, or in any case, what are people's thoughts on deprecating them? Maybe not right away but eventually, just to avoid config drift? Especially if it's unlikely they were ever used in the wild (unless I'm mistaken about that)

Possibly nothing to do here (unless we know for sure they could be deleted) but could raise an issue if we think that's a good idea.